sli/dictutils.h

```diff
@@ -251,6 +251,36 @@ updateValue( DictionaryDatum const& d, Name const n, VT& value )
   value = getValue< FT >( t );
   return true;
 }
+/* Polymorphic version of update_value.
+ * This code will take either an int or a double and convert it to an
+ * int.
+ */
+bool
```

Comment:
You need an `inline` before `bool` to make the code compile. This is why the Github actions fail. So, it needs to be `inline bool` here.